packages/fxa-settings/src/components/Settings/PageRecoveryKeyCreate/index.tsx

```diff
@@ -2,49 +2,130 @@
 import FlowRecoveryKeyInfo from '../FlowRecoveryKeyInfo';
+import VerifiedSessionGuard from '../VerifiedSessionGuard';
+
+const viewName = 'settings.account-recovery';
 
 export const PageRecoveryKeyCreate = (props: RouteComponentProps) => {
-  const navigate = useNavigate();
+  usePageViewEvent(viewName);
+
+  const account = useAccount();
   const ftlMsgResolver = useFtlMsgResolver();
+  const navigate = useNavigate();
+
+  const [currentStep, setCurrentStep] = useState<number>(1);
+  const [formattedRecoveryKey, setFormattedRecoveryKey] = useState<string>('');
+
   const goHome = () => navigate(HomePath + '#recovery-key', { replace: true });
-  const [currentStep, setCurrentStep] = useState(1);
-  usePageViewEvent('settings.account-recovery');
+  const numberOfSteps = 4;
+
   const localizedPageTitle = ftlMsgResolver.getMsg(
     'recovery-key-create-page-title',
     'Account Recovery Key'
   );
 
-  /*
-    The content here will obviously be replaced as we complete the separate views for this flow. This page will use the same pattern as the example wizard in storybook for `FlowContainer`. All steps will be separate components which use the `FlowContainer` and accept `currentStep` and `setCurrentStep` as the props necessary to move the user through the flow.
-  */
+  const localizedBackButtonTitle = ftlMsgResolver.getMsg(
+    'recovery-key-create-back-button-title',
+    'Back to settings'
+  );
+
+  const navigateBackward = () => {
+    navigate('/settings');
+  };
+
+  const navigateForward = (e?: React.MouseEvent<HTMLElement>) => {
+    e?.preventDefault();
+    if (currentStep + 1 <= numberOfSteps) {
+      setCurrentStep(currentStep + 1);
+    } else {
+      navigate('/settings');
+    }
+  };
+
+  // Redirects to settings if a recovery key is already set
+  // TODO change to redirect to 'PageRecoveryKeyChange' once created
+  useEffect(() => {
+    if (account.recoveryKey && !formattedRecoveryKey) {
+      navigate(HomePath, { replace: true });
+    }
+  }, [account, formattedRecoveryKey, navigate]);
+
+  // TODO prevent page refresh? currently refreshing breaks user from flow
+
   return (
     <>
       <VerifiedSessionGuard onDismiss={goHome} onError={goHome} />
-      {
-        // Create an account recovery key
-        currentStep === 1 && (
-          <FlowRecoveryKeyInfo
-            {...{ localizedPageTitle }}
-            navigateForward={() => {
-              setCurrentStep(2);
-            }}
-            navigateBackward={() => {
-              navigate('/settings');
+      {/* Switch through the account recovery key steps based on step number */}
+      {/* Create an account recovery key */}
+      {currentStep === 1 && (
+        <FlowRecoveryKeyInfo
+          {...{
+            currentStep,
+            numberOfSteps,
+            localizedBackButtonTitle,
+            localizedPageTitle,
+            navigateBackward,
+            navigateForward,
+            viewName,
+          }}
```

Comment:
These seem duplicated in almost every step.
Not sure if we want to extract them a bit higher into a variable which we can reuse.


```js
const stepProps = {
            currentStep,
            numberOfSteps,
            localizedBackButtonTitle,
            localizedPageTitle,
            navigateBackward,
            navigateForward,
            viewName,
          };

<FlowRecoveryKeyInfo {...{
    ...flowProps
}} />
<FlowRecoveryKeyConfirmPwd {...{
    ...flowProps,
    setFormattedRecoveryKey,
}} />
<FlowRecoveryKeyDownload {...{
    ...flowProps,
    recoveryKeyValue: formattedRecoveryKey,
}} />
```

Not sure if that works, or is any cleaner/better. But might be a bit DRYer and easier if we ever change shared props.